src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java

```diff
@@ -75,11 +74,11 @@ public StormTopology createTopology() {
 
         OpenTsdbConfig openTsdbConfig = topologyConfig.getOpenTsdbConfig();
 
-        declareBolt(tb, new DatapointParseBolt(), OTSDB_PARSE_BOLT_ID)
+        declareBolt(tb, new DatapointParseBolt(ZooKeeperSpout.SPOUT_ID), OTSDB_PARSE_BOLT_ID)
                 .shuffleGrouping(OTSDB_SPOUT_ID)
                 .allGrouping(ZooKeeperSpout.SPOUT_ID);
 
-        declareBolt(tb, new OpenTSDBFilterBolt(), OTSDB_FILTER_BOLT_ID)
+        declareBolt(tb, new OpenTsdbFilterBolt(ZooKeeperSpout.SPOUT_ID), OTSDB_FILTER_BOLT_ID)
```

Comment:
You added active/inactive state for this bolt but:
1. You didn't added stream from this bolt to ZkBolt
2. I think we shouldn't add Zk spout here because OpenTsdbFilterBolt receives tuples from DatapointParseBolt. But DatapointParseBolt already has active/inactive states